broadlink/climate.py

```diff
@@ -460,34 +487,31 @@ def set_advanced(self,
             speed_L, speed_R = 0x20, 0x00
         elif (speed == 'mute'):
             speed_L, speed_R = 0x40, 0x80
-            assert (mode == 'F')
+            if mode != 'fan':
+                raise ValueError('mute speed is only available in fan mode')
         elif (speed == 'turbo'):
-            speed_R = 0x40
             speed_L = 0x20 # doesn't matter
+            speed_R = 0x40
+            if not (mode == 'cooling' or mode == 'heating'):
```

Comment:
A much more readable way to do it:
```python3
if mode not in ('cooling', 'heating'):
```

Reply:
Any reason why tuples over lists?

Reply:
tuples are faster (I think)
https://stackoverflow.com/questions/1708510/list-vs-tuple-when-to-use-each

Reply:
Thanks for teaching me that!